CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.1.5] - 2024-05-22
+### Fixed
+- Add `copy_tags_to_snapshot` to aws_rds_cluster.
```

Comment:
I suspect the previous change to provider version made this a setting with default to false. At least that's what the latest version has: https://registry.terraform.io/providers/hashicorp/aws/5.50.0/docs/resources/rds_cluster#copy_tags_to_snapshot
Older version docs are not available.

So overriding cause all our clusters have this by default to true.